modules/icress_module.php

```diff
@@ -77,26 +79,30 @@ function icress_getCampus($campus, $faculty) {
 			if ($key === 0) {
 				continue;
 			}
-			$subject = $row->childNodes[3]->nodeValue;
-			$subjects[] = array($subject);
+			$subject = rtrim($row->childNodes[3]->nodeValue);
```

Comment:
The `path` I mentioned previously is located inside a `button`

It looks like this:

` <button id="btn300" onClick="window.open('index_tt.cfm?id1=7620E16BD38FB6E7A9CB28E5A81A918EBCC6&id2=701DDF471CB3B558EC09C2DD505EA4671387')" class="btn btn-success btn-block waves-effect waves-light" type="button">View</button>`